pkgs/applications/networking/cluster/cmctl/default.nix

```
@@ -2,37 +2,53 @@
 
 buildGoModule rec {
   pname = "cmctl";
-  version = "1.8.2";
+  version = "1.9.1";
 
   src = fetchFromGitHub {
     owner = "cert-manager";
-    repo = "cert-manager";
-    rev = "v${version}";
-    sha256 = "sha256-sfC1acnCrcQ4A1tXXcjh47Af6xeJqjdGXy0gK21ZSFg=";
+    repo  = "cert-manager";
+    rev   = "v${version}";
+    hash  = "sha256-Z1aJ18X4mfJPlCPBC7QgfdX5Tk4+PK8mYoJZhGwz9ec=";
   };
 
-  vendorSha256 = "sha256-UYw9WdQ6VwzuuiOsa1yovkLZG7NmLYSW51p8UhmQMeI=";
+  vendorSha256 = "sha256-45+tZZAEHaLdTN1NQCueJVTx5x2IanwDl+Y9MELqdBE=";
 
   subPackages = [ "cmd/ctl" ];
 
-  ldflags = [ "-s" "-w" ];
+  ldflags = [
+    "-s" "-w"
+    # See https://github.com/cert-manager/cert-manager/blob/4486c01f726f17d2790a8a563ae6bc6e98465505/make/cmctl.mk#L1
+    "-X" "github.com/cert-manager/cert-manager/cmd/ctl/pkg/build.name=cmctl"
+    "-X" "github.com/cert-manager/cert-manager/cmd/ctl/pkg/build/commands.registerCompletion=true"
+  ];
 
   nativeBuildInputs = [ installShellFiles ];
 
+  # Ensure we capture runtime completions by calling `completion` subcommand.
   postInstall = ''
     mv $out/bin/ctl $out/bin/cmctl
-    installShellCompletion --cmd cmctl \
-      --bash <($out/bin/cmctl completion bash) \
-      --fish <($out/bin/cmctl completion fish) \
-      --zsh <($out/bin/cmctl completion zsh)
+    $out/bin/cmctl completion bash > cmctl.bash
+    $out/bin/cmctl completion zsh  > cmctl.zsh
+    $out/bin/cmctl completion fish > cmctl.fish
+    installShellCompletion cmctl.{bash,zsh,fish}
   '';
 
   meta = with lib; {
-    description = "A CLI tool for managing Cert-Manager service on Kubernetes clusters";
+    description = "A CLI tool for managing cert-manager service on Kubernetes clusters";
+    longDescription = ''
+      cert-manager adds certificates and certificate issuers as resource types
+      in Kubernetes clusters, and simplifies the process of obtaining, renewing
+      and using those certificates.
+
+      It can issue certificates from a variety of supported sources, including
+      Let's Encrypt, HashiCorp Vault, and Venafi as well as private PKI, and it
+      ensures certificates remain valid and up to date, attempting to renew
+      certificates at an appropriate time before expiry.
+    '';
     downloadPage = "https://github.com/cert-manager/cert-manager";
     license = licenses.asl20;
     homepage = "https://cert-manager.io/";
-    maintainers = with maintainers; [ superherointj ];
+    maintainers = with maintainers; [ superherointj joshvanl ];
```

```suggestion
    maintainers = with maintainers; [ joshvanl superherointj ];
```
Ideally we should follow alphabetical order on lists. (Same applies to other fields.)